scripts/generate-examples.py

```diff
@@ -111,6 +110,10 @@
                 metaOptions = json.loads(meta)
                 options.update(metaOptions)
 
+            # If the example has additional options, load them.
+            example_options: str = example.get("options", "{}")
```

Comment:
If you define this in YAML as a dictionary, you don't have to do the JSON string operations; you can load this and do the update directly: 

```
example_options: Dict = example.get("options", {})
options.update(example_options);
```